governator-legacy/src/test/java/com/netflix/governator/lifecycle/LifeCycleFeaturesOnLegacyBuilderTest.java

```diff
@@ -0,0 +1,69 @@
+package com.netflix.governator.lifecycle;
+
+import static org.junit.Assert.assertNotNull;
+
+import javax.annotation.PostConstruct;
+import javax.annotation.PreDestroy;
+
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.mockito.Mockito;
+import org.mockito.Spy;
+import org.mockito.runners.MockitoJUnitRunner;
+
+import com.google.inject.AbstractModule;
+import com.google.inject.Injector;
+import com.netflix.governator.LifecycleManager;
+import com.netflix.governator.guice.LifecycleInjector;
+import com.netflix.governator.spi.LifecycleListener;
+
+@RunWith(MockitoJUnitRunner.class)
+public class LifeCycleFeaturesOnLegacyBuilderTest {
+    
+    private com.netflix.governator.lifecycle.LifecycleManager legacyLifecycleManager;
+
+    @Spy
+    private final TestListener listener = new TestListener();
```

Comment:
I didn't care about the behavior of this class since all of the methods are no-ops. So a spy seemed logical.